invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/deposits/RDMDepositForm/RDMDepositForm.js

```diff
@@ -157,13 +228,14 @@ export class RDMDepositForm extends Component {
                   options={this.vocabularies.metadata.creators}
                   required
                 />
+                <PublicationDateField required />
+
                 <ContributorsField
                   options={this.vocabularies.metadata.contributors}
                 />
                 <DescriptionsField
                   options={this.vocabularies.metadata.descriptions}
                 />
-                <PublicationDateField required />
```

Comment:
The mockup here: https://github.com/inveniosoftware/product-rdm/issues/3 puts `PublicationDateField` below Contributors and Description.

Reply:
Fixed. Set according to mockup.